org.lflang/src/org/lflang/generator/c/CCmakeCompiler.java

```diff
@@ -108,6 +112,11 @@ public boolean runCCompiler(
         // Make sure the build directory exists
         Files.createDirectories(buildPath);
 
+        // If we are running an Arduino Target, need to copy over the BoardOptions file.
+        if(targetConfig.platform == Platform.ARDUINO) {
+            FileUtil.copyFile(FileUtil.globFilesEndsWith(fileConfig.srcPath, "BoardOptions.cmake").get(0), Paths.get(buildPath.toString(), File.separator, "BoardOptions.cmake"));
```

Comment:
I don't think copying files should happen here. It should probably happen around [here](https://github.com/lf-lang/lingua-franca/blob/8b35b5937c1529fa9ac36bb34ae1d9d7497c74eb/org.lflang/src/org/lflang/generator/c/CGenerator.java#L567).

Reply:
Fixed.